feature-libs/checkout/b2b/components/checkout-review-submit/checkout-review-submit.component.ts

```diff
@@ -71,22 +71,10 @@ export class B2BCheckoutReviewSubmitComponent extends CheckoutReviewSubmitCompon
     return this.checkoutPaymentTypeFacade.isAccountPayment();
   }
 
-  /**
-   *  TODO:#checkout - since the `getCostCenterState` returns the actual cost center,
```

Comment:
Without the episode, it makes sense to use the super query as it contains the current set cost center for the user's cart